============
File Formats
============
Below are the `FileFormat <object_model/types/file_format.html>`_ values currently supported by SMaHT Portal.

.. toctree::
  :caption: File Formats
  :maxdepth: 1


For AlignedReads
----------------
.. raw:: html

  <table class="schema-table" width="104%"> <tr> <th width="5%"> Name </th> <th width="30%"> ID </th> <th width="65%"> Description <a target="_blank" href="https://data.smaht.org/search/?type=FileFormat&valid_item_types=AlignedReads"><span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;color:white;" /></a></span></th> </tr> <tr> <td> <b>bam</b> </td> <td> <a target="_blank" href="https://data.smaht.org/d13d06cf-218e-4f61-aaf0-91f226248b3c">d13d06cf-218e-4f61-aaf0-91f226248b3c<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Binary version of a SAM file. Format used to represent aligned sequences.<br /><u>File Extension</u>: <b>.bam</b><br /><u>Valid Types</u>: <a href='object_model/types/aligned_reads.html'>AlignedReads</b>, <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b>, <a href='object_model/types/unaligned_reads.html'>UnalignedReads</b> </td> </tr> <tr> <td> <b>cram</b> </td> <td> <a target="_blank" href="https://data.smaht.org/d363c5f9-7159-45b1-b516-e5ec433f9b86">d363c5f9-7159-45b1-b516-e5ec433f9b86<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Compressed version of a BAM file. Format used to represent aligned sequences.<br /><u>File Extension</u>: <b>.cram</b><br /><u>Valid Types</u>: <a href='object_model/types/aligned_reads.html'>AlignedReads</b>, <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> </table><p />


For UnalignedReads
------------------
.. raw:: html

  <table class="schema-table" width="104%"> <tr> <th width="5%"> Name </th> <th width="30%"> ID </th> <th width="65%"> Description <a target="_blank" href="https://data.smaht.org/search/?type=FileFormat&valid_item_types=UnalignedReads"><span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;color:white;" /></a></span></th> </tr> <tr> <td> <b>bam</b> </td> <td> <a target="_blank" href="https://data.smaht.org/d13d06cf-218e-4f61-aaf0-91f226248b3c">d13d06cf-218e-4f61-aaf0-91f226248b3c<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Binary version of a SAM file. Format used to represent aligned sequences.<br /><u>File Extension</u>: <b>.bam</b><br /><u>Valid Types</u>: <a href='object_model/types/aligned_reads.html'>AlignedReads</b>, <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b>, <a href='object_model/types/unaligned_reads.html'>UnalignedReads</b> </td> </tr> <tr> <td> <b>fastq_gz</b> </td> <td> <a target="_blank" href="https://data.smaht.org/c13d06cf-218e-4f61-aaf0-91f226248b2c">c13d06cf-218e-4f61-aaf0-91f226248b2c<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent short read sequence data, compressed.<br /><u>File Extension</u>: <b>.fastq.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b>, <a href='object_model/types/unaligned_reads.html'>UnalignedReads</b> </td> </tr> </table><p />


For VariantCalls
-----------------
.. raw:: html

  <table class="schema-table" width="104%"> <tr> <th width="5%"> Name </th> <th width="30%"> ID </th> <th width="65%"> Description <a target="_blank" href="https://data.smaht.org/search/?type=FileFormat&valid_item_types=VariantCalls"><span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;color:white;" /></a></span></th> </tr> <tr> <td> <b>vcf</b> </td> <td> <a target="_blank" href="https://data.smaht.org/fcc2647d-301b-4888-8d9d-83ea4270309c">fcc2647d-301b-4888-8d9d-83ea4270309c<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent genomics variants.<br /><u>File Extension</u>: <b>.vcf</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b>, <a href='object_model/types/variant_calls.html'>VariantCalls</b> </td> </tr> <tr> <td> <b>vcf_gz</b> </td> <td> <a target="_blank" href="https://data.smaht.org/1b8f525f-aecb-4211-9ae5-a2c998b05599">1b8f525f-aecb-4211-9ae5-a2c998b05599<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Compressed version of a VCF file. Format used to represent genomics variants.<br /><u>File Extension</u>: <b>.vcf.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b>, <a href='object_model/types/variant_calls.html'>VariantCalls</b> </td> </tr> </table><p />


All File Formats
----------------
.. raw:: html

  <table class="schema-table" width="104%"> <tr> <th width="5%"> Name </th> <th width="30%"> ID </th> <th width="65%"> Description <a target="_blank" href="https://data.smaht.org/search/?type=FileFormat"><span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;color:white;" /></a></span></th> </tr> <tr> <td> <b>Rdata</b> </td> <td> <a target="_blank" href="https://data.smaht.org/ce424ef5-86c8-4522-aecf-6c1c98f365b5">ce424ef5-86c8-4522-aecf-6c1c98f365b5<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent R objects.<br /><u>File Extension</u>: <b>.Rdata</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>alt</b> </td> <td> <a target="_blank" href="https://data.smaht.org/9ed3e9f9-fee2-47e3-bbe3-c63a52f8d3b8">9ed3e9f9-fee2-47e3-bbe3-c63a52f8d3b8<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to BWT.<br /><u>File Extension</u>: <b>.alt</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>amb</b> </td> <td> <a target="_blank" href="https://data.smaht.org/8db70ed6-0121-4fe1-a72e-d91dc5aa6cd3">8db70ed6-0121-4fe1-a72e-d91dc5aa6cd3<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to BWT.<br /><u>File Extension</u>: <b>.amb</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>ann</b> </td> <td> <a target="_blank" href="https://data.smaht.org/106199e5-5a85-4817-9a55-7b31698e1fda">106199e5-5a85-4817-9a55-7b31698e1fda<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to BWT.<br /><u>File Extension</u>: <b>.ann</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>bam</b> </td> <td> <a target="_blank" href="https://data.smaht.org/d13d06cf-218e-4f61-aaf0-91f226248b3c">d13d06cf-218e-4f61-aaf0-91f226248b3c<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Binary version of a SAM file. Format used to represent aligned sequences.<br /><u>File Extension</u>: <b>.bam</b><br /><u>Valid Types</u>: <a href='object_model/types/aligned_reads.html'>AlignedReads</b>, <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b>, <a href='object_model/types/unaligned_reads.html'>UnalignedReads</b> </td> </tr> <tr> <td> <b>bam_bai</b> </td> <td> <a target="_blank" href="https://data.smaht.org/d13d06c1-218e-4f61-aaf0-91f226248b3c">d13d06c1-218e-4f61-aaf0-91f226248b3c<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to BAM. Format used to represent the index of a BAM file.<br /><u>File Extension</u>: <b>.bam.bai</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>bed</b> </td> <td> <a target="_blank" href="https://data.smaht.org/4c04f6de-89a7-4477-8dc4-811b50c67401">4c04f6de-89a7-4477-8dc4-811b50c67401<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> BED (Browser Extensible Data) format is a text file format used to store genomic regions as coordinates and associated annotations.<br /><u>File Extension</u>: <b>.bed</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>bed_gz</b> </td> <td> <a target="_blank" href="https://data.smaht.org/4f074eca-29a0-4a49-b335-aef988e6cbd7">4f074eca-29a0-4a49-b335-aef988e6cbd7<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Compressed version of a BED file. Format used to store genomic regions as coordinates and associated annotations.<br /><u>File Extension</u>: <b>.bed.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>bed_gz_tbi</b> </td> <td> <a target="_blank" href="https://data.smaht.org/40346690-6359-4436-97ff-562698ab4b31">40346690-6359-4436-97ff-562698ab4b31<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to compressed BED. Format used to represent the index of a compressed BED file (Tabix generated).<br /><u>File Extension</u>: <b>.bed.gz.tbi</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>big</b> </td> <td> <a target="_blank" href="https://data.smaht.org/f66af4df-c107-44f0-bc94-227f1b4ccf72">f66af4df-c107-44f0-bc94-227f1b4ccf72<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent a binary index for the genome.<br /><u>File Extension</u>: <b>.big</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>bigWig</b> </td> <td> <a target="_blank" href="https://data.smaht.org/33f30c42-d582-4163-af44-fecf586b9dd3">33f30c42-d582-4163-af44-fecf586b9dd3<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Binary version of a Wig file. Format used for display of dense continuous data with genomic coordinates.<br /><u>File Extension</u>: <b>.bw</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>bwt</b> </td> <td> <a target="_blank" href="https://data.smaht.org/813b0001-5f3f-4e28-9203-4cdf261e19c4">813b0001-5f3f-4e28-9203-4cdf261e19c4<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent the genome index based on Burrows-Wheeler Transform (BWT).<br /><u>File Extension</u>: <b>.bwt</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>chain</b> </td> <td> <a target="_blank" href="https://data.smaht.org/dd1ef82d-da5e-4680-bd5c-cf471a87eb5b">dd1ef82d-da5e-4680-bd5c-cf471a87eb5b<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent pairwise alignment that allow gaps in both sequences simultaneously, compressed.<br /><u>File Extension</u>: <b>.chain.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>contamination</b> </td> <td> <a target="_blank" href="https://data.smaht.org/d0cba8b5-cd01-41f5-bfed-e5369293d2dd">d0cba8b5-cd01-41f5-bfed-e5369293d2dd<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> TXT format to report metrics generated by Sentieon ContaminationModel algorithm.<br /><u>File Extension</u>: <b>.contamination</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>cram</b> </td> <td> <a target="_blank" href="https://data.smaht.org/d363c5f9-7159-45b1-b516-e5ec433f9b86">d363c5f9-7159-45b1-b516-e5ec433f9b86<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Compressed version of a BAM file. Format used to represent aligned sequences.<br /><u>File Extension</u>: <b>.cram</b><br /><u>Valid Types</u>: <a href='object_model/types/aligned_reads.html'>AlignedReads</b>, <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>dbnsfp_gz</b> </td> <td> <a target="_blank" href="https://data.smaht.org/65a2cca2-dae8-4ff2-ac8b-aa1e92f5416b">65a2cca2-dae8-4ff2-ac8b-aa1e92f5416b<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format to represent the dbNSFP database as a compressed VCF.<br /><u>File Extension</u>: <b>.dbnsfp.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>dbnsfp_gz_tbi</b> </td> <td> <a target="_blank" href="https://data.smaht.org/311ac7bf-e1d5-463f-af15-61ebfea29608">311ac7bf-e1d5-463f-af15-61ebfea29608<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to compressed dbNSFP. Format used to represent the index of a compressed dbNSFP file (Tabix generated).<br /><u>File Extension</u>: <b>.dbnsfp.gz.tbi</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>dbnsfp_readme_txt</b> </td> <td> <a target="_blank" href="https://data.smaht.org/ac822ea4-d281-41e0-b9c9-f283c51f1c15">ac822ea4-d281-41e0-b9c9-f283c51f1c15<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to compressed dbNSFP. Format used to store dbNSFP README as plain text.<br /><u>File Extension</u>: <b>.dbnsfp.readme.txt</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>dict</b> </td> <td> <a target="_blank" href="https://data.smaht.org/4ed9f7e0-2b2f-4aca-9533-a0a652b43442">4ed9f7e0-2b2f-4aca-9533-a0a652b43442<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to FASTA.<br /><u>File Extension</u>: <b>.dict</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>fa</b> </td> <td> <a target="_blank" href="https://data.smaht.org/5ced774b-a73e-4d1b-8186-d7fbbde7a3c2">5ced774b-a73e-4d1b-8186-d7fbbde7a3c2<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> FASTA format. Format used to represent the genome reference sequence.<br /><u>File Extension</u>: <b>.fa</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>fa_fai</b> </td> <td> <a target="_blank" href="https://data.smaht.org/fb728bb4-bc56-46d5-8df5-a05562826b9a">fb728bb4-bc56-46d5-8df5-a05562826b9a<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to FASTA.<br /><u>File Extension</u>: <b>.fa.fai</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>fastq</b> </td> <td> <a target="_blank" href="https://data.smaht.org/eb417c0a-70dd-42e3-9841-ac7f1ee22962">eb417c0a-70dd-42e3-9841-ac7f1ee22962<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> <u>File Extension</u>: <b>.fastq</b> </td> </tr> <tr> <td> <b>fastq_gz</b> </td> <td> <a target="_blank" href="https://data.smaht.org/c13d06cf-218e-4f61-aaf0-91f226248b2c">c13d06cf-218e-4f61-aaf0-91f226248b2c<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent short read sequence data, compressed.<br /><u>File Extension</u>: <b>.fastq.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b>, <a href='object_model/types/unaligned_reads.html'>UnalignedReads</b> </td> </tr> <tr> <td> <b>gff3</b> </td> <td> <a target="_blank" href="https://data.smaht.org/f87864e0-7d55-46bd-a67a-fb8753ce87db">f87864e0-7d55-46bd-a67a-fb8753ce87db<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> GFF (General Feature Format) Version 3, used for storing genomic features as a text file.<br /><u>File Extension</u>: <b>.gff3</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>gvcf</b> </td> <td> <a target="_blank" href="https://data.smaht.org/f592a45e-3b8a-4bad-bfd4-52acf9fd663d">f592a45e-3b8a-4bad-bfd4-52acf9fd663d<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent genomics variant sites, GVCF has records for all sites (whether there is a variant call there or not).<br /><u>File Extension</u>: <b>.gvcf</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>gvcf_gz</b> </td> <td> <a target="_blank" href="https://data.smaht.org/ad47d469-4561-4234-bce2-820f08f58e7c">ad47d469-4561-4234-bce2-820f08f58e7c<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Compressed version of a GVCF file. Format used to represent genomics variant sites.<br /><u>File Extension</u>: <b>.gvcf.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>gvcf_gz_tbi</b> </td> <td> <a target="_blank" href="https://data.smaht.org/b01ee86e-b2c7-4725-81d7-798718674485">b01ee86e-b2c7-4725-81d7-798718674485<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to compressed GVCF. Format used to represent the index of a compressed GVCF file (Tabix generated).<br /><u>File Extension</u>: <b>.gvcf.gz.tbi</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>json</b> </td> <td> <a target="_blank" href="https://data.smaht.org/ff12517a-d51e-45a4-8f44-a1cfe418dba5">ff12517a-d51e-45a4-8f44-a1cfe418dba5<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent JavaScript Object Notation (JSON).<br /><u>File Extension</u>: <b>.json</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>md5_list</b> </td> <td> <a target="_blank" href="https://data.smaht.org/1362126e-e6ee-4010-9fb8-06e9b39dbb83">1362126e-e6ee-4010-9fb8-06e9b39dbb83<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format to represent the list of contigs MD5 produced by cramtools getref command.<br /><u>File Extension</u>: <b>.md5_list</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>pac</b> </td> <td> <a target="_blank" href="https://data.smaht.org/7373ca48-0b3e-467b-967a-80870846f89b">7373ca48-0b3e-467b-967a-80870846f89b<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to BWT.<br /><u>File Extension</u>: <b>.pac</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>pdf</b> </td> <td> <a target="_blank" href="https://data.smaht.org/81b7ce7f-64ed-4933-96d1-b6df498a7664">81b7ce7f-64ed-4933-96d1-b6df498a7664<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent Portable Document Format (PDF).<br /><u>File Extension</u>: <b>.pdf</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>plugins_tar</b> </td> <td> <a target="_blank" href="https://data.smaht.org/65ccbf65-80f9-41b4-b002-468500821c62">65ccbf65-80f9-41b4-b002-468500821c62<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to VEP archive. Format used to represent VEP plugins as archive, compressed.<br /><u>File Extension</u>: <b>.plugins.tar.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>png</b> </td> <td> <a target="_blank" href="https://data.smaht.org/7c525767-e142-45f6-b4c3-84f52bc6f4cc">7c525767-e142-45f6-b4c3-84f52bc6f4cc<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> PNG (Portable Graphics Format). Format used to represent an uncompressed image.<br /><u>File Extension</u>: <b>.png</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>priors</b> </td> <td> <a target="_blank" href="https://data.smaht.org/c4f4538f-ff79-42f0-a3be-d416251475ae">c4f4538f-ff79-42f0-a3be-d416251475ae<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> TXT format to report metrics generated by Sentieon OrientationBias algorithm.<br /><u>File Extension</u>: <b>.priors</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>rck</b> </td> <td> <a target="_blank" href="https://data.smaht.org/228190b1-4178-46ad-872e-9b8ca1931a31">228190b1-4178-46ad-872e-9b8ca1931a31<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> RCK (Read Count Keeper) format, used to store pileup read counts by strand and allele.<br /><u>File Extension</u>: <b>.rck</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>rck_gz</b> </td> <td> <a target="_blank" href="https://data.smaht.org/20d4d3aa-5f1c-4b75-9e25-73f9f370fefa">20d4d3aa-5f1c-4b75-9e25-73f9f370fefa<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Compressed version of RCK file. Format used to store pileup read counts by strand and allele.<br /><u>File Extension</u>: <b>.rck.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>rck_gz_tbi</b> </td> <td> <a target="_blank" href="https://data.smaht.org/c55ace88-3289-49b0-a67a-c046e1004e5a">c55ace88-3289-49b0-a67a-c046e1004e5a<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to compressed RCK. Format used to represent the index of a compressed RCK file (Tabix generated).<br /><u>File Extension</u>: <b>.rck.gz.tbi</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>rck_tar</b> </td> <td> <a target="_blank" href="https://data.smaht.org/39f836d8-bbb1-46c7-80d4-e321d4a44204">39f836d8-bbb1-46c7-80d4-e321d4a44204<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent an archive of compressed RCK files.<br /><u>File Extension</u>: <b>.rck.tar</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>rck_tar_index</b> </td> <td> <a target="_blank" href="https://data.smaht.org/1c7dc723-811c-4fcf-b8e5-d5e17a2013f7">1c7dc723-811c-4fcf-b8e5-d5e17a2013f7<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to RCK archive. Format used to represent the index of files in the archive.<br /><u>File Extension</u>: <b>.rck.tar.index</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>sa</b> </td> <td> <a target="_blank" href="https://data.smaht.org/11f2fc36-9a65-4d60-9365-d8ff241df4f7">11f2fc36-9a65-4d60-9365-d8ff241df4f7<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to BWT.<br /><u>File Extension</u>: <b>.sa</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>sam</b> </td> <td> <a target="_blank" href="https://data.smaht.org/3311fb05-a0df-43e5-b0af-234c82b6bee9">3311fb05-a0df-43e5-b0af-234c82b6bee9<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent aligned sequences.<br /><u>File Extension</u>: <b>.sam</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>tar</b> </td> <td> <a target="_blank" href="https://data.smaht.org/39866342-e4f8-4a50-87bf-b61a782549c8">39866342-e4f8-4a50-87bf-b61a782549c8<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent an archive of files.<br /><u>File Extension</u>: <b>.tar</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>tar_gz</b> </td> <td> <a target="_blank" href="https://data.smaht.org/f2ec3b9f-a898-4e6c-8da5-734a7a6410b8">f2ec3b9f-a898-4e6c-8da5-734a7a6410b8<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Compressed version of a TAR archive. Format used to represent an archive of files.<br /><u>File Extension</u>: <b>.tar.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>tsv</b> </td> <td> <a target="_blank" href="https://data.smaht.org/c369d5d6-2861-47ab-bc39-99083cfe48bd">c369d5d6-2861-47ab-bc39-99083cfe48bd<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent Tab-Separate Values (TSV).<br /><u>File Extension</u>: <b>.tsv</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>tsv_gz</b> </td> <td> <a target="_blank" href="https://data.smaht.org/11ca3783-db6e-430e-997b-9cf0ca275814">11ca3783-db6e-430e-997b-9cf0ca275814<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Compressed version of a TSV file. Format used to represent Tab-Separate Values.<br /><u>File Extension</u>: <b>.tsv.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>tsv_gz_tbi</b> </td> <td> <a target="_blank" href="https://data.smaht.org/829ed303-e427-4d9a-a217-be75ad11317e">829ed303-e427-4d9a-a217-be75ad11317e<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to compressed TSV. Format used to represent the index of a compressed TSV file (Tabix generated).<br /><u>File Extension</u>: <b>.tsv.gz.tbi</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>txt</b> </td> <td> <a target="_blank" href="https://data.smaht.org/0cd4e777-a596-4927-95c8-b07716121aa3">0cd4e777-a596-4927-95c8-b07716121aa3<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent plain text.<br /><u>File Extension</u>: <b>.txt</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>vcf</b> </td> <td> <a target="_blank" href="https://data.smaht.org/fcc2647d-301b-4888-8d9d-83ea4270309c">fcc2647d-301b-4888-8d9d-83ea4270309c<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent genomics variants.<br /><u>File Extension</u>: <b>.vcf</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b>, <a href='object_model/types/variant_calls.html'>VariantCalls</b> </td> </tr> <tr> <td> <b>vcf_gz</b> </td> <td> <a target="_blank" href="https://data.smaht.org/1b8f525f-aecb-4211-9ae5-a2c998b05599">1b8f525f-aecb-4211-9ae5-a2c998b05599<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Compressed version of a VCF file. Format used to represent genomics variants.<br /><u>File Extension</u>: <b>.vcf.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b>, <a href='object_model/types/variant_calls.html'>VariantCalls</b> </td> </tr> <tr> <td> <b>vcf_gz_stats</b> </td> <td> <a target="_blank" href="https://data.smaht.org/ec465f66-f1ae-44e0-9885-a2b24e7ce268">ec465f66-f1ae-44e0-9885-a2b24e7ce268<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to compressed VCF. Format used to collect metrics for a compressed VCF file.<br /><u>File Extension</u>: <b>.vcf.gz.stats</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>vcf_gz_tbi</b> </td> <td> <a target="_blank" href="https://data.smaht.org/f84f1922-7f4e-4afc-922f-bec620969bf1">f84f1922-7f4e-4afc-922f-bec620969bf1<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to compressed VCF. Format used to represent the index of a compressed VCF file (Tabix generated).<br /><u>File Extension</u>: <b>.vcf.gz.tbi</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>vcf_idx</b> </td> <td> <a target="_blank" href="https://data.smaht.org/ec96f95a-cf13-4633-ab0d-c4a5138bbe0b">ec96f95a-cf13-4633-ab0d-c4a5138bbe0b<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Companion format to VCF. Format used to represent the index of a VCF file.<br /><u>File Extension</u>: <b>.vcf.idx</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>vcf_tar</b> </td> <td> <a target="_blank" href="https://data.smaht.org/3d140fc3-fd0e-4d09-8294-4536e388e665">3d140fc3-fd0e-4d09-8294-4536e388e665<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used to represent an archive of compressed VCF files.<br /><u>File Extension</u>: <b>.vcf.tar</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>vep_tar</b> </td> <td> <a target="_blank" href="https://data.smaht.org/d05f9688-0ee1-4a86-83f4-656e6e21352a">d05f9688-0ee1-4a86-83f4-656e6e21352a<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format to represent VEP datasource as archive, compressed.<br /><u>File Extension</u>: <b>.vep.tar.gz</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>wig</b> </td> <td> <a target="_blank" href="https://data.smaht.org/19e290b5-2743-4311-a860-5dfca41858b1">19e290b5-2743-4311-a860-5dfca41858b1<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Format used for display of dense continuous data with genomic coordinates.<br /><u>File Extension</u>: <b>.wig</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> <tr> <td> <b>zip</b> </td> <td> <a target="_blank" href="https://data.smaht.org/1125243b-3acd-4793-9264-4abd7d788e58">1125243b-3acd-4793-9264-4abd7d788e58<span class="fa fa-external-link" style="left:4pt;position:relative;top:1pt;" /></a> </td> <td> Archive file format that supports lossless data compression.<br /><u>File Extension</u>: <b>.zip</b><br /><u>Valid Types</u>: <a href='object_model/types/output_file.html'>OutputFile</b>, <a href='object_model/types/reference_file.html'>ReferenceFile</b> </td> </tr> </table><p />
